test/plugins/individual-collateral/morpho-aave/MorphoAAVEFiatCollateral.test.ts

```diff
@@ -0,0 +1,226 @@
+    }
+
+    const collateral = await deployCollateral(collateralOpts)
+
+    return {
+      alice,
+      collateral,
+      underlyingErc20: underlyingErc20,
+      chainlinkFeed,
+      tok: wrapperMock as unknown as ERC20Mock,
+      morphoWrapper: wrapperMock,
+    } as MorphoAaveCollateralFixtureContext
+  }
+
+  return makeCollateralFixtureContext
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const reduceTargetPerRef = async () => {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const increaseTargetPerRef = async () => {}
+
+const changeRefPerTok = async (
+  ctx: MorphoAaveCollateralFixtureContext,
+  percentChange: BigNumber
+) => {
+  const rate = await ctx.morphoWrapper.getExchangeRate()
+  await ctx.morphoWrapper.setExchangeRate(rate.add(rate.mul(percentChange).div(bn('100'))))
+}
+
+// prettier-ignore
+const reduceRefPerTok = async (
+  ctx: MorphoAaveCollateralFixtureContext,
+  pctDecrease: BigNumberish
+) => {
+  await changeRefPerTok(
+    ctx,
+    bn(pctDecrease).mul(-1)
+  )
+}
+// prettier-ignore
+const increaseRefPerTok = async (
+  ctx: MorphoAaveCollateralFixtureContext,
+  pctIncrease: BigNumberish
+) => {
+  await changeRefPerTok(
+    ctx,
+    bn(pctIncrease)
+  )
+}
+const getExpectedPrice = async (ctx: MorphoAaveCollateralFixtureContext): Promise<BigNumber> => {
+  const clData = await ctx.chainlinkFeed.latestRoundData()
+  const clDecimals = await ctx.chainlinkFeed.decimals()
+
+  const refPerTok = await ctx.collateral.refPerTok()
+  return clData.answer
+    .mul(bn(10).pow(18 - clDecimals))
+    .mul(refPerTok)
+    .div(fp('1'))
+}
+
+/*
+  Define collateral-specific tests
+*/
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const collateralSpecificConstructorTests = () => {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const collateralSpecificStatusTests = () => {}
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const beforeEachRewardsTest = async () => {}
+
+export const defaultCollateralOpts: MAFiatCollateralOpts = {
+  targetName: ethers.utils.formatBytes32String('USDT'),
+  underlyingToken: networkConfig[1].tokens.USDT!,
+  poolToken: networkConfig[1].tokens.aUSDT!,
+  priceTimeout: PRICE_TIMEOUT,
+  chainlinkFeed: USDC_USD_PRICE_FEED,
+  oracleTimeout: ORACLE_TIMEOUT,
+  oracleError: ORACLE_ERROR,
+  maxTradeVolume: bn(1000000),
+  defaultThreshold: DEFAULT_THRESHOLD,
+  delayUntilDefault: DELAY_UNTIL_DEFAULT,
+  revenueHiding: fp('0'),
+  defaultPrice: bn('1e8'),
+  defaultRefPerTok: fp('1'),
+}
+
+/*
+  Run the test suite
+*/
+
+const opts = {
+  deployCollateral,
+  collateralSpecificConstructorTests,
+  collateralSpecificStatusTests,
+  beforeEachRewardsTest,
+  makeCollateralFixtureContext,
+  mintCollateralTo,
+  reduceTargetPerRef,
+  increaseTargetPerRef,
+  reduceRefPerTok,
+  increaseRefPerTok,
+  getExpectedPrice,
+  itClaimsRewards: it.skip,
```

Comment:
I think it's worth having a specific test for the proof-style claiming -- can probably grab a claim that occurred after this fork block and use that